### PR TITLE
[DDW-569] Move ITN redemption to Daedalus application menu

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR moved ITN redemption to Daedalus application menu
[Jira Ticket](https://jira.iohk.io/browse/DDW-569)

## Todos
- [x] Move ITN redemption to Daedalus application menu

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1614191965023900)

### App
- [x] Click on menu item `Daedalus` => `Redeem ITN rewards` and make sure ITN redemption dialog is shown.

### English
image

### Japanese
image

### Test Cases
```Gherkin
Scenario 1 - Button greyed out and inactive before I accept terms and conditions
Given that I have Daedalus installed
And I launch Daedalus
When I am yet to agree to the Terms and Conditions
Then the Daedalus menu item for its rewards is greyed out and not active

Scenario 2 - Button active after I accept terms and conditions
Given that I launched Daedalus 
And I have accepted the Terms and Conditions
When I open the Daedalus menu items
Then I see that the button is active
And when I click on it I get brought to the itn rewards redemption screen
```

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by ManusMcCole** [comment]
Hi @yakovkaravelov. Tested on build Mainnet 16676. Functionality works well although I found if the user changes the language to Japanese the button reverts back to being greyed out and inactive. See <a href="https://drive.google.com/file/d/1xeL3oKlmC5XFx3Wqv0cTHWh-J9HldzSA/view?usp=sharing">video</a>. If Daedalus is closed and reopened the button is active again

**Comment by nikolaglumac** [comment]
> Hi @yakovkaravelov. Tested on build Mainnet 16676. Functionality works well although I found if the user changes the language to Japanese the button reverts back to being greyed out and inactive. See [video](https://drive.google.com/file/d/1xeL3oKlmC5XFx3Wqv0cTHWh-J9HldzSA/view?usp=sharing). If Daedalus is closed and reopened the button is active again

Good catch @ManusMcCole! This is obviously a bug that most likely affects develop and production builds too.
It affects ALL the links as far as I can see...
@yakovkaravelov please fix this 🙏

File: source/renderer/app/stores/SidebarStore.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ export default class SidebarStore extends Store {
       [categories.PAPER_WALLET_CREATE_CERTIFICATE.name]: false,
       [categories.STAKING_DELEGATION_COUNTDOWN.name]: false,
       [categories.STAKING.name]: true,
-      [categories.REDEEM_ITN_REWARDS.name]: true,
+      [categories.REDEEM_ITN_REWARDS.name]: false,
```
Comment (nikolaglumac) [inline source/renderer/app/stores/SidebarStore.js:86]: @yakovkaravelov please remove this line completely and clean up any leftovers (e.g. categories).

**Comment by nikolaglumac** [comment]
@yakovkaravelov please merge latest develop.

**Comment by yakovkaravelov** [comment]
> > Hi @yakovkaravelov. Tested on build Mainnet 16676. Functionality works well although I found if the user changes the language to Japanese the button reverts back to being greyed out and inactive. See [video](https://drive.google.com/file/d/1xeL3oKlmC5XFx3Wqv0cTHWh-J9HldzSA/view?usp=sharing). If Daedalus is closed and reopened the button is active again
> 
> Good catch @ManusMcCole! This is obviously a bug that most likely affects develop and production builds too.
> It affects ALL the links as far as I can see...
> @yakovkaravelov please fix this 🙏

This bus was not introduced by this PR actually. It is already existing in develop branch as well, anyway, will fix it.
CC @nikolaglumac @ManusMcCole

**Comment by nikolaglumac** [comment]
@yakovkaravelov please fix it regardless of it's origin.

**Comment by yakovkaravelov** [comment]
@tomislavhoracek @nikolaglumac @daniloprates @miorsufianiohk @ManusMcCole Please review/test, thanks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ Changelog
 
 ### Chores
 
+- Moved ITN redemption to Daedalus application menu ([PR 2401](https://github.com/input-output-hk/daedalus/pull/2401))
+- Adjusted hover PopOver styles on the first tile in the delegation center screen ([PR 2386](https://github.com/input-output-hk/daedalus/pull/2386))
```
Comment (tomislavhoracek) [inline CHANGELOG.md:24]: @yakovkaravelov why is this here?

**Review by tomislavhoracek — CHANGES_REQUESTED** [review]
@yakovkaravelov I left one comment. Can you please take a look?
And one more question. Why you removed `isUpdateAvailable` checks?

File: CHANGELOG.md (hunk 2 above)
Comment (yakovkaravelov) [reply CHANGELOG.md:24]: @tomislavhoracek `isUpdateAvailable` was removed after verifying that it is not used anymore by Nikola.

Comment (yakovkaravelov) [reply CHANGELOG.md:24]: This changelog issue was generated by some git merge issue. Will fix it.

**Review by tomislavhoracek — APPROVED** [review]

**Review by ManusMcCole — APPROVED** [review]
Looks good @yakovkaravelov

**Review by miorsufianiohk — APPROVED** [review]
Looks good to me. Tested on 16833. Good job @yakovkaravelov 👍

**Comment by darko-mijic** [comment]
Reviewing this now

**Review by darko-mijic — APPROVED** [review]

**Review by gabriela-ponce — CHANGES_REQUESTED** [review]
@yakovkaravelov The screen can be accessed while Daedalus is loading and [it looks like it hasn't any wallet](https://drive.google.com/file/d/1Juu2Du6IY2I-VVaqEGmjKSjIzSA-9oVY/view?usp=sharing). It changes after the loading is complete, but I think it looks odd. Can this be improved? For reference, the settings and wallet settings screens can't be accessed in that state.

**Comment by nikolaglumac** [comment]
> @yakovkaravelov The screen can be accessed while Daedalus is loading and [it looks like it hasn't any wallet](https://drive.google.com/file/d/1Juu2Du6IY2I-VVaqEGmjKSjIzSA-9oVY/view?usp=sharing). It changes after the loading is complete, but I think it looks odd. Can this be improved? For reference, the settings and wallet settings screens can't be accessed in that state.

Good point @gabriela-ponce - I have modified the logic so that we show the "syncing" status message instead of no-wallets in case Daedalus is not fully synced.

Screenshot 2021-02-27 at 14 50 11

**Review by nikolaglumac — APPROVED** [review]
Looks good!
